setup.py

```diff
@@ -0,0 +1,7 @@
+'''
+EC Note: updated versions of pip do NOT require a setup.py file but older versions will, so I'm including just in case :)
+'''
```

Comment:
let's exclude this and enforce using an updated pip

Reply:
Done! Removed setup.py from the file directory. It shouldn't cause any errors, but I saved the setup.py code elsewhere (locally) just in case.